src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -98,6 +99,13 @@ private FileVisitResult actuallyVisitFile(Path inPath, BasicFileAttributes attrs
                 } else {
                     Files.copy(inPath, outPath);
                 }
+
+                if(outPath.toString().endsWith(".class")) { 
+                    Files.write(
+                        outFS.getPath((outPath.toString() + "-paths-for-coverage.txt")),
```

Comment:
There's something I'm not sure I understand.
IIUC `srcs` contains the list of all sources for the jar we're processing, right?
At the same time `outPath` represents a single `.class` file (in the output jar but I'm not sure it's important), right?
If so why do we add another txt file per each .class file and in the .txt write all of the sources?

Reply:
Ah! You are correct. I was incorrectly making an assumption off of a comment in bazel which read:

```
If a jar contains uninstrumented classes it will also contain a txt file with the paths of each of these classes, one on each line.
```

To which I interpreted as must have a corresponding `-paths-for-coverage.txt`, but that is incorrect, it just needs a single file named as such.

Reply:
I see.
My main concern then is that no test failed when the implementation actually didn't work, right?
Or do both versions work?
And finally- if I change the filename to `-paths-blah-for-coverage.txt` will any test break? Can we add such a test? I think it can be super valuable.

Reply:
Both implementations did actually work. 

What happens in bazel/jacoco is quite crude. It essentially scans all JARs for `-paths-for-coverage.txt` and adds them all to a single set. Regardless of what jar it comes from. 

Yes, if you do change to that file name it will fail (see the two commits I just pushed).